### PR TITLE
pointer cursor for volume slider

### PR DESCRIPTION
**Pull request by realdiegopoptart**

### CODE REVIEW

File: pages/sa.vue [hunk 1]
```diff
@@ -638,6 +638,7 @@ section.radio {
                     $base: 15%;
                     $bars: 16;
 
+                    cursor: url("@/assets/images/cursors/sa-pointer.png"), pointer;;
```
Comment (CommandString) [inline pages/sa.vue:641]: Remove extra `;`

File: pages/sa.vue [hunk 2]
```diff
@@ -646,7 +647,6 @@ section.radio {
                     &--muted {
                         pointer-events: none;
                     }
-
```
Comment (CommandString) [inline pages/sa.vue:649]: Add this line back

**Review by CommandString — CHANGES_REQUESTED** [review]

File: pages/sa.vue (hunk 2 above)
Comment (realdiegopoptart) [reply pages/sa.vue:649]: i was about to fix that lmaoooo, i dont even remember doing that

File: pages/sa.vue (hunk 1 above)
Comment (realdiegopoptart) [reply pages/sa.vue:641]: i think one of my extentions ranomly added it lol
\

**Pull request merged**